.travis.yml

```diff
@@ -15,4 +15,9 @@ before_install:
   - yarn install
 install:
   - tsc -p .
-  - docker-compose up
+  - docker-compose build
+after_success:
+  - if [ "$TRAVIS_BRANCH" == "develop" ]; then 
+    docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"; 
+    docker push musicconnectionmachine/unstructureddata; 
```

Comment:
Does this automatically push to docker hub? I don't see this repo there yet. Maybe we need another commit?